Network Observability 1.7 regenerate flows format doc (#83465)
openshift/openshift-docs: modules/network-observability-flows-format.adoc

The "Loki label" column is useful when querying Loki directly: label fields need to be selected using link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].

> The "Cardinality" column gives information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.

Reviewer: ```suggestion
The "Cardinality" column contains information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. For more information about that API, see the `FlowMetrics` documentation.
```
The OCP documentation guidelines seem to prefer the "For more information, see ..." wording for pointing to other content.

Author: Good nits, thank you!
